downstream/modules/automation-hub/proc-obtaining-org-collection-url.adoc

```diff
@@ -13,8 +13,6 @@ You can synchronize {CertifiedName} curated by your organization from `{Console}
 
 . Log in to `{Console}` as an organization administrator.
 . Navigate to menu:Automation Hub[Repo Management].
-. Locate the *Sync URL* and click the btn:[Copy to clipboard] icon (image:copy.png[Copy,15,20]). 
```

Comment:
When the docs say `Automation Hub[Repo Management]` and `Automation Hub[Repositories]` I'm not sure if that refers to console.redhat.com Hub (CRC) or Private Automation Hub (PAH).

CRC will not have any menu items for "Repo Management" or "Repositories", you can see the pre-prod code here https://console.stage.redhat.com/ansible/automation-hub/

PAH no longer has a "Repo Management" menu item, and if you want to paste a repo URL from CRC it will be under the "Remotes" menu item. (the rh-certified remote already has the URL for CRC certified content pre-populated, so unless it was manually changed earlier there is no need to paste). PAH screenshot:

![image](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/assets/4432895/578cf97d-a2c6-4bd6-a45b-06e0258f773b)